### ISSUE TITLE
Gene glyphs are not rendering

### ISSUE THREAD

**Issue by shashankbrgowda** [GMOD/Apollo3#273]
We are not rendering any glyph if the parent feature is 'gene' and its sub-feature is not equal to 'mRNA'

example gff
```
ctgA	example	gene	1050	9000	.	+	.	ID=EDEN;Name=EDEN;Note=protein kinase
ctgA	example	V_gene_segment	1050	9000	.	+	.	ID=EDEN.1;Parent=EDEN;Name=EDEN.1;Note=Eden splice form 1;Index=1
ctgA	example	five_prime_UTR	1050	1200	.	+	.	Parent=EDEN.1
ctgA	example	CDS	1201	1500	.	+	0	Parent=EDEN.1
ctgA	example	CDS	3000	3902	.	+	0	Parent=EDEN.1
ctgA	example	CDS	5000	5500	.	+	0	Parent=EDEN.1
ctgA	example	CDS	7000	7608	.	+	0	Parent=EDEN.1
ctgA	example	three_prime_UTR	7609	9000	.	+	.	Parent=EDEN.1
```

**Comment by shashankbrgowda** [comment]
![image](https://github.com/GMOD/Apollo3/assets/13611226/ba0b1a53-c8de-4a81-b7c6-bdc6a0b930b3)

**Comment by garrettjstevens** [comment]
Just to clarify the expected behavior here, since "V_gene_segment" is not a valid child in the ontology of "gene", this should use the fallback glyph and not a gene glyph.

**Comment by dariober** [comment]
Also the GFF below doesn't show any glyph even if it has gene -> mRNA -> exon

![image](https://github.com/GMOD/Apollo3/assets/6883382/72d098ba-63b6-4e82-a6cf-9102a01ffb1d)


```
##gff-version 3
ctgA	example	gene	1	100	.	+	.	ID=gx1
ctgA	example	mRNA	1	100	.	+	.	ID=tx1;Parent=gx1
ctgA	example	exon	1	100	.	+	.	ID=exon1;Parent=tx1
ctgA	example	gene	100	200	.	+	.	ID=gx2
ctgA	example	mRNA	100	200	.	+	.	ID=tx2;Parent=gx2
ctgA	example	exon	100	200	.	+	.	ID=exon2;Parent=tx2
##FASTA
>ctgA
cattgttgcggagttgaacaACGGCATTAGGAACACTTCCGTCTCtcacttttatacgat
tatgattggttctttagccttggtttagattggtagtagtagcggcgctaatgctacctg
aattgagaactcgagcgggggctaggcaaattctgattcagcctgacttctcttggaacc
ctgcccataaatcaaagggttagtgcggccaaaacgttggacaacggtattagaagacca
acctgaccaccaaaccgtcaattaaccggtatcttctcggaaacggcggttctctcctag
atagcgatctgtggtctcaccatgcaatttaaacaggtgagtaaagattgctacaaatac
gagactagctgtcaccagatgctgttcatctgttggctccttggtcgctccgttgtaccc
aggctactttgaaagagcgcagaatacttagacggtatcgatcatggtagcatagcattc
tgataacatgtatggagttcgaacatccgtctggggccggacggtccgtttgaggttggt
tgatctgggtgatagtcagcaagatagacgttagataacaaattaaaggattttacctta
gattgcgactagtacaacggtacatcggtgattcgcgctctactagatcacgctatgggt
accataaacaaacggtggaccttctcaagctggttgacgcctcagcaacataggcttcct
cctccacgcatctcagcataaaaggcttataaactgcttctttgtgccagagcaactcaa
ttaagcccttggtaccgtgggcacgcattctgtcacggtgaccaactgttcatcctgaat
cgccgaatgggactatttggtacaggaatcaagcggatggcactactgcagcttatttac
gacggtattcttaaagtttttaagacaatgtatttcatgggtagttcggtttgttttatt
gctacacaggctcttgtagacgacctacttagcactacggccgagcgcaataacccccgg
aaagcacttgctactgggaggcgggtttatccatcggcaataggggttatcagtactacc
aagaagattgtgaagatattaacagcattgaaaaaagttcggactgggcatgaaacgtgt
```

**Comment by shashankbrgowda** [comment]
CDS is not rendered when mRNA has exons and CDS doesn't contain discontinuous location.

When a feature contain exon, glyph is rendered using CanonicalGeneGlyph. CanonicalGeneGlyph doesn't render CDS if it doesn't contain discontinuous locations.

![image](https://github.com/GMOD/Apollo3/assets/13611226/eb65d1cf-b216-46ce-93a9-4820b966c08e)

**Comment by shashankbrgowda** [comment]
User request: Ability to render operon - union of genes. It seems like operon is not rendered in jbrowse at all and apollo track renders it as GenericChildGlyph.

Apollo track:
![image](https://github.com/GMOD/Apollo3/assets/13611226/7c7635ed-179d-4f12-bcc5-9813ac80eb36)

Jbrowse:
![image](https://github.com/GMOD/Apollo3/assets/13611226/ce4afa40-e3f7-44eb-a07f-de34ee9f4f5a)

**Comment by garrettjstevens** [comment]
Gene glyph rendering fixed in #412, and operon rendering now tracked in #440
